### DISCUSSION TITLE
How can I make x axis of nivo/line bold

### DISCUSSION THREAD

**Discussion by vishalkak** [plouc/nivo#1596] [Q&A] [answered]
I have create a nivo line chart that looks like this

![Capture](https://user-images.githubusercontent.com/42554215/122181537-5b222700-ce81-11eb-9e89-1cc20473e336.JPG)

How can I make x-axis line bold at 0

**Comment by wyze** [comment, marked as answer]
You can customize the `renderTick` function of `axisLeft`. It will look similar to the following:

```ts
const MyLineChart = () => <Line axisLeft={{ renderTick: (value) => <text style={{ fontWeight: value === 0 ? 'bold' : 'normal'}}>{value}</text> }} />
```
